ci/run_truffle_tests.sh

```diff
@@ -38,8 +38,8 @@ run_tests() {
     # Check the ci deployment.
     check_deployment ./ 1234 ci
 
-    # Run standard truffle tests
-    $(npm bin)/truffle test --network ci
+    # Run standard buidler tests
+    $(npm bin)/buidler test
```

Comment:
Just to be on same page, making this change (and the change in `coverage.sh`) will *replace* truffle-test with buidler, right?

Reply:
Yes it will replace `truffle-test` with buidler on the ci.

I wanted to test it in the existing ci pipeline to see how much improvement it will provide a realistic scenario. I can revert it back if its too much.

Reply:
This SGTM. @nicholaspai @chrismaree thoughts?

Reply:
Oh, sorry, I meant replacing truffle test with buidler test sounds good to me since it’ll speed up ci considerably. +1. Just wanted to double check that @nicholaspai and @chrismaree were also cool with it.